primeqa/tableqa/tapex/README.md

```diff
@@ -0,0 +1,78 @@
+# Table Question Answering using TAPEX
+
+TAPEX is a table pre-training approach for table-related tasks. By learning a neural SQL executor over a synthetic corpus based on generative language models (e.g., BART), it achieves state-of-the-art performance on several table question answering and table fact verification benchmarks. More details can be found in the original paper [TAPEX: Table Pre-training via Learning a Neural SQL Executor](https://arxiv.org/pdf/2107.07653.pdf).
+
+PrimeQA supports TAPEX based training and eval over two major table question answering datasets like wikisql, wikitablequestions
```

Comment:
PrimeQA supports `TAPEX-based` training

Reply:
This indeed means based on Tapex not tapex-base.